src/platform/auth.ts

```diff
@@ -35,6 +36,15 @@ export const userHasPermission = (
 export const comparePassword = (password: string, hash: string) =>
 	sbvrUtils.sbvrTypes.Hashed.compare(password, hash);
 
+/**
+ * A known invalid comparisson to emulate a wrong password error
```

Comment:
Can you add to the comment that it's in order to prevent exposing information via timing please - the comment states *what* it does, but not *why* it's important

Reply:
Added @Page- :+1: 
Wasn't sure if we were fine having such reasoning in the open-balena-api repo, otherwise I would have targeted for the why for sure, instead of the what.